libs/ledger-live-common/src/account/serialization.ts

```diff
@@ -109,7 +109,7 @@ export function toBalanceHistoryRaw(b: BalanceHistory): BalanceHistoryRaw {
 export function fromBalanceHistoryRaw(b: BalanceHistoryRaw): BalanceHistory {
   return b.map(([date, value]) => ({
     date: new Date(date),
-    value: new BigNumber(value),
+    value: new BigNumber(value).toNumber(),
```

Comment:
I'm actually scared to use a something else like a parseFloat there... any insight on this ?

Reply:
Discussed with @haammar-ledger, moved to parseFloat